docroot/modules/contrib/ova/ova.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Open Video Annotation using template variables.
+ */
+
+ /**
+ *  Implements template_preprocess_hook().
+ */
+function template_preprocess_ovaAudio(&$variables) {
+  $annotation_settings = array();
```

Comment:
This settings we are using in addannotatebutton.js file. Yes helper.js file is not required. we will remove it.